json-json4s/shared/src/main/scala/rapture/json-json4s/ast.scala

```diff
@@ -78,7 +77,10 @@ private[json4s] object Json4sAst extends JsonBufferAst {
 
   def setObjectValue(obj: Any, name: String, value: Any): Any = {
     val contents = (name, value) :: obj.asInstanceOf[JObject].obj.filter(_._1 != name)
-    JObject(contents map { case (k: String, v: JValue) => k -> v })
+    JObject(contents map {
+      case (k: String, v: JValue) => k -> v
+      case _ => ???
```

Comment:
The only one reason why I have added it because `match may not exhaustive` warning. "-Xfatal-warnings" translate it to error. 
So we have a few options:
1. Add `case _ => ???`
2. Just add `@unchecked` annotation 
3.  Remove "-Xfatal-warnings" 

Maybe the second option would be preferred?